examples/darwin-framework-tool/commands/pairing/DeviceControllerDelegateBridge.mm

```diff
@@ -22,7 +22,7 @@
 @implementation CHIPToolDeviceControllerDelegate
 - (void)controller:(MTRDeviceController *)controller statusUpdate:(MTRCommissioningStatus)status
 {
-    NSLog(@"Pairing Status Update: %tu", status);
+    NSLog(@"Pairing Status Update: %ld", (long) status);
```

Comment:
static_cast, please.  Same elsewhere in this PR.

And should be `%lu` and `unsigned long`, not `long`.  Also, same elsewhere in this PR.

Reply:
Nope, this is:

typedef NS_ENUM(NSInteger, MTRCommissioningStatus) {
    MTRCommissioningStatusUnknown = 0,
    MTRCommissioningStatusSuccess = 1,
    MTRCommissioningStatusFailed = 2,
    MTRCommissioningStatusDiscoveringMoreDevices MTR_DEPRECATED("MTRCommissioningStatusDiscoveringMoreDevices is not used.",
        ios(16.1, 16.5), macos(13.0, 13.4), watchos(9.1, 9.5), tvos(16.1, 16.5))
    = 3,


Which is not unsigned.

Reply:
Fixing static casts though.